src/features/jserrors/instrument/index.js

```diff
@@ -84,6 +84,8 @@ export class Instrument extends InstrumentBase {
    * @returns
    */
   onerrorHandler (message, filename, lineno, column, errorObj) {
+    if (typeof this.origOnerror === 'function') this.origOnerror(...arguments)
```

Comment:
I think this will need to be reverted, because if the customer defines their own handler to return `true` (ie to block the error from reaching the console) we should honor that.

Reply:
>if the customer defines their own handler to return `true`

You mean if they edit `window.onerror`, and not via `setErrorHandler(func)`? 
This is still a function:
```javascript
window.onerror = function() { return true }
```

Reply:
Yes if they edit onerror before the agent is set, we should honor their return value (true/false)

What you have is always returning false

Reply:
ok in that case we will just keep an eye on this and iterate in the future if customers need a way to return "true" out of the top-level `onerror` method :+1: